# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1208983812 14400
#      Wed Apr 23 16:50:12 2008 -0400
# Branch release-3-0-x
# Node ID 6a8fd365eebd29ace49c6c86f1be67471467d087
# Parent  af3f3c6f381f95fbf02349cc98067584801de146
Added tag release-3-0-1 for changeset 57504aa0461e

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -30,9 +30,10 @@ 10da0f6d85c260bb5f640436e42d633a57ccda9b
 db6d908e8a03a42760a45c7fd29eae074e505f7c ss-2-9-12
 afbd31bb7b4e001c6c16d41eb908aa66bed64352 ss-2-9-13
 fc55a5e1760b832a672d9de3b53888a3d6793d21 ss-2-9-14
 0b91144f9533b05d14fb5692ed67d8c6f500230f ss-2-9-15
 5608935ff5228a90d34fa80dce3dab2462d20fd3 ss-2-9-16
 77de8319c337b39a477d5433348e4597ad61abe0 ss-2-9-17
 9f02c32eb70d3e6d0c42a2fd5f3b862f4d6c6bcc ss-2-9-18
 89d546610556ee47ec634ab8c3e441e1db248873 ss-2-9-19
+57504aa0461e08a1771f3c66f0e6b6759d564887 release-3-0-1
 274d9642ac1032fcc4996a3cac80cfe5fd20f8a4 release-3-0-0
